### ISSUE TITLE
Show charging/discharging state in battery indicator

### ISSUE THREAD

**Issue by vswarte** [vswarte/gostatus#9]
Indicate the current state of the battery's charge/discharge. Maybe display a plug when charging?

**Comment by AuditeMarlow** [comment]
Charging: :zap:

**Comment by vswarte** [comment]
Since we have moved away from unicode emoji in favor of configurable labels it would be wise to reserve two configuration options.
I suggest we reserve the following two properties:
```
"label_discharging":　"DISCHARGING",
"label_charging": "CHARGING"
```

**Comment by vswarte** [comment]
Partially implemented per d3b4830

**Comment by vswarte** [comment]
Fully implemented
